backend/server/src/http/mod.rs

```diff
@@ -170,50 +170,56 @@ async fn handle(req: Request<Body>, ctx: Arc<Context>) -> Result<Response> {
         req.uri().query().map(|q| format!("?{}", q)).unwrap_or_default(),
     );
 
-    let method = req.method();
+    let method = req.method().clone();
     let path = req.uri().path();
 
     const ASSET_PREFIX: &str = "/assets/";
     const REALM_PREFIX: &str = "/r/";
     const PLAYER_PREFIX: &str = "/v/";
 
-    let response = match (method, path) {
-        (&Method::GET, "/") | (&Method::GET, "/about") => ctx.assets.serve_index().await,
+    let response = match path {
+        // The GraphQL endpoint. This is the only path for which POST is
+        // allowed.
+        "/graphql" if method == Method::POST => {
+            juniper_hyper::graphql(ctx.api_root.clone(), ctx.api_context.clone(), req).await?
+        }
+
+        // From this point on, we only support GET and HEAD requests. All others
+        // will result in 404.
+        _ if method != Method::GET && method != Method::HEAD => {
+            reply_404(&ctx.assets, &method, path).await
+        }
```

Comment:
Shouldn't this return [405](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405)? :thinking: